templates/emails/transactional/invalid-totp-login.html

```diff
@@ -1,6 +1,7 @@
 {% extends "base.html" %}
 
 {% block content %}
+  {{ render_text("Hello," ~ (" " ~ name ~ "," if name)) }}
```

Comment:
Hey I think this is actually harder to read than using `name = user.name or ""` :)

Reply:
Right, but if we used that then the text would be "Hello, ," if the user.name was None.

Reply:
That's right. For this reason in other emails we only put "Hi" without the name :). As most SL users don't set the name field, this option seems to be enough.

Reply:
Yup, but as outlined in [these comments](https://github.com/simple-login/app/pull/753#issuecomment-1022096915), it's good to include something SimpleLogin would know in these emails.

If readability is that much of an issue, we could just expand it into an if/else, for example:
HTML
```jinja2
{% if name %}
  {{ render_text("Hello, " ~ name ~ ",") }}
{% else %}
  {{ render_text("Hello,") }}
{% endif %}
```

Plaintext
```jinja2
{% if name %}
Hello, {{ name }},
{% else %}
Hello,
{% endif %}
```

Reply:
Sorry to not participate in the discussion earlier but I don't think including the user.name is really helpful as most users don't have their name set and other emails don't use this information neither. So I think the current TOTP warning emails are already good enough :).

Reply:
@nguyenkims Alright, feel free to close my PRs earlier in the future if you feel like this, I'm not going to be offended ;)

Reply:
Thanks :)